src/templates/common/identifiers/crossref_journal_issue.xml

```diff
@@ -7,6 +7,12 @@
                 {% if crossref_issue.journal.journal_issn and crossref_issue.journal.journal_issn != '0000-0000' %}
                 <issn media_type="electronic">{{ crossref_issue.journal.journal_issn }}</issn>
                 {% endif %}
+                {% if crossref_issue.journal.doi %}
+                <doi_data>
+                    <doi>{{ crossref_issue.journal.doi }}</doi>
+                    <resource>{{ crosref_issue.journal.url }}</resource>
+                </doi_data>
+                {% endif %}
```

Comment:
The `journal` here is different from the standard journal object. It is created by `create_crossref_journal_context` above (in order to handle custom titles in ISSNs). So the `doi` attribute will need to be added there before it shows up in this template.